proto/agglayer/interop/types/v1/aggchain.proto

```diff
@@ -8,18 +8,57 @@ import "agglayer/interop/types/v1/bytes.proto";
 message AggchainData {
   // The data of the aggchain.
   oneof data {
-    // Signature committed to the bridge exits and imported bridge exits.
+    // Legacy ECDSA signature on a commitment to the state transition.
     FixedBytes65 signature = 1;
-    // Generic Aggchain proof.
+
+    // Aggchain proof with the single ECDSA verified by the agglayer only.
     AggchainProof generic = 2;
+
+    // Only multisig performed by the chain's committee.
+    Multisig multisig = 3;
+
+    // Multisig combined with an aggchain proof.
+    AggchainProofWithMultisig multisig_and_aggchain_proof = 4;
+  }
+}
+
+// Multisig
+message Multisig {
+  // Multisig scheme.
+  oneof data {
+    // ECDSA multisig scheme.
+    ECDSAMultisig ecdsa = 1;
   }
 }
 
+// ECDSA multisig scheme with signatures from the committee members.
+message ECDSAMultisig {
+  // Represents the key-value pair of a committee member signature entry.
+  message ECDSAMultisigEntry {
+    // The key is the place of the signer in the list of multisig signer on L1.
+    uint64 key = 1;
```

Comment:
sorry just occurred to me that we probably want to align all references to threshold and signer index to a u32 given that in the L1 we'll never have more than 255 signers, but that's not blocking, we can do the change in next rounds